rs/rust_canisters/canister_test/src/canister.rs

```diff
@@ -807,17 +793,29 @@ impl<'a> Canister<'a> {
         let stop_res: Result<(), String> = self
             .runtime
             .get_management_canister_with_effective_canister_id(self.canister_id().into())
-            .update_("stop_canister", candid_multi_arity, (self.as_record(),))
+            .update_(
+                "stop_canister",
+                candid_multi_arity,
+                (StopCanisterArgs {
+                    canister_id: self.canister_id().into(),
+                },),
```

Comment:
Can this be simplified to just

```suggestion
                candid_one,
                StopCanisterArgs {
                    canister_id: DestinationType:from(self.canister_id()),
                },
```

?

Reply:
Done.